### PR TITLE
64524 tests refactoring

### PR DESCRIPTION
**Pull request by SergeyMul**

### CODE REVIEW

File: test/webdriverio/specs/jira.js [hunk 1]
```diff
@@ -0,0 +1,237 @@
+describe("Jira", function () {
+
+  var bugTrackerUrl = 'https://jira.atlassian.com';
+
+  var testProjectName = 'Demo';
```
Comment (alexserdyuk) [inline test/webdriverio/specs/jira.js]: Why two constants?

File: test/webdriverio/specs/jira.js [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+        .click('#full-issue-navigator a')
+        .waitForExist('.save-as-new-filter')
+        .click('.save-as-new-filter')
+        .waitForExist('#filterName')
+        .setValue('#filterName', testFilterName)
+        .click('.submit')
+        .waitForExist('.search-title=' + testFilterName);
+    }
+
+    function createScrumBoard () {
```
Comment (alexserdyuk) [inline test/webdriverio/specs/jira.js]: Leave only one board with less code.

File: test/webdriverio/customCommands.js [hunk 3]
```diff
@@ -55,14 +58,11 @@ browser.addCommand("startAndTestTaskStarted", function (projectName, taskName, t
     .getAttribute('.timer-active a.flex-item-no-shrink', 'href').should.eventually.be.equal(taskUrl)
```
Comment (ArtemAvramenko) [inline test/webdriverio/customCommands.js]: Avoid complicated selectors. I add 'issue-link' class to work item.

File: test/webdriverio/specs/jira.js [hunk 4]
```diff
@@ -0,0 +1,173 @@
+describe("Jira", function () {
+
+  var bugTrackerUrl = 'https://jira.atlassian.com';
```
Comment (alexserdyuk) [inline test/webdriverio/specs/jira.js]: remove constants that are used only once. Move constants to test script for better readability

File: test/webdriverio/specs/jira.js [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+
+  var testProjectName = 'Demo';
+  var testProjectUrl = bugTrackerUrl + '/projects/DEMO';
+
+  var testIssueName = 'Issue-qweasdzxc for ' + testProjectName;
```
Comment (alexserdyuk) [inline test/webdriverio/specs/jira.js]: No need to add suffixes. use literals

File: test/webdriverio/specs/jira.js [hunk 6]
```diff
@@ -0,0 +1,173 @@
...
+
+  before(function () {
+
+    var testIssueAnchorSelector = '//a[span[contains(@class,"issue-link-summary")][text()="' + testIssueName + '"]]';
+    var testKanbanBoardAnchorSelector = '//a[text()="' + testKanbanBoardName + '"]';
+
+    function createIssue () {
+      return browser
+        .url(testIssueSearchUrl)
+        .waitForExist('#create_link')
```
Comment (alexserdyuk) [inline test/webdriverio/specs/jira.js]: Create 
client.waitAndClick(selector).then(callback);
command for shortening the scripts

**Comment by ArtemAvramenko** [comment]
Import type definitions for chai and chai-as-promised. Remove jasmine.d.ts

**Pull request merged**